Tests/IntegrationTests/Roslyn/Resources/RuntimeDependencies.pom

```diff
@@ -16,6 +16,12 @@
     </dependency>
     
     <dependency>
+      <groupId>org.codehaus.sonar.sslr-squid-bridge</groupId>
+      <artifactId>sslr-squid-bridge</artifactId>
+      <version>2.6</version>
+    </dependency>
```

Comment:
Jars that embed sqale files need this dependency to execute (contains the SqaleXmlLoader class)